GhSA/Helpers/PostHog.cs

```diff
@@ -18,11 +18,19 @@ public static async Task<HttpResponseMessage> SendToPostHog(string eventName, Di
     {
       // posthog ADS plugin requires a user object
       User user = new User();
-      user.email = UserPrincipal.Current.EmailAddress; // case sensitive
-
       string userName = Environment.UserName;
-      if (!user.email.EndsWith("arup.com"))
-        userName = "External";
+      try
+      {
+        user.email = UserPrincipal.Current.EmailAddress; // case sensitive
+        
+        if (!user.email.EndsWith("arup.com"))
+          userName = "External";
+      }
+      catch (Exception)
```

Comment:
what exception are you trying to catch here?

Reply:
reverted